### ISSUE TITLE
Can't access settings 

### ISSUE THREAD

**Issue by arthuragone** [nextgis/quickmapservices#141]
When trying to go to settings I got this error:

```
2017-10-13T16:08:09	1	Traceback (most recent call last):
			  File "/Users/arthur/.qgis2/python/plugins/quick_map_services/quick_map_services.py", line 299, in show_settings_dialog
			    settings_dlg = SettingsDialog()
			  File "/Users/arthur/.qgis2/python/plugins/quick_map_services/settings_dialog.py", line 48, in __init__
			    self.setupUi(self)
			  File "<string>", line 119, in setupUi
			  File "/Users/arthur/.qgis2/python/plugins/quick_map_services/user_groups_box.py", line 26, in __init__
			    self.feel_list()
			  File "/Users/arthur/.qgis2/python/plugins/quick_map_services/user_groups_box.py", line 44, in feel_list
			    ds_groups = GroupsList(USER_GROUP_PATHS)
			  File "/Users/arthur/.qgis2/python/plugins/quick_map_services/groups_list.py", line 55, in __init__
			    self.locale = Locale.get_locale()
			  File "/Users/arthur/.qgis2/python/plugins/quick_map_services/locale.py", line 30, in get_locale
			    return QSettings().value('locale/userLocale')[0:2]
			TypeError: 'QPyNullVariant' object has no attribute '__getitem__'
```

**Comment by simgislab** [comment]
version of QGIS, version of QMS?

**Comment by arthuragone** [comment]
QGIS 2.18.13 (OSX)
QMS: 0.19.3

**Comment by 514ckw4r3** [comment]
Fixed in version 0.19.24. The fix will be available for download in the next release.
